addon/globalPlugins/remoteClient/__init__.py

```diff
@@ -100,8 +100,11 @@ def create_menu(self):
 		self.disconnect_item.Enable(False)
 		gui.mainFrame.sysTrayIcon.Bind(wx.EVT_MENU, self.on_disconnect_item, self.disconnect_item)
 		# Translators: Menu item in NvDA Remote submenu to mute speech and sounds from the remote computer.
-		self.mute_item = self.menu.Append(wx.ID_ANY, _("Mute remote"), _("Mute speech and sounds from the remote computer"))
-		self.mute_item.SetCheckable(True)
+		if dialogs.WX_VERSION<=3:
+			self.mute_item = self.menu.Append(wx.ID_ANY, _("Mute remote"), _("Mute speech and sounds from the remote computer"))
+			self.mute_item.SetCheckable(True)
+		else:

```

Comment:
This works on the WX Python 3.0 I have, so the first part of this if is unneeded.